src/devices/Cargo.toml

```diff
@@ -5,12 +5,15 @@ authors = ["The Chromium OS Authors"]
 edition = "2018"
 
 [dependencies]
-epoll = ">=4.0.1"
 libc = ">=0.2.39"
 dumbo = { path = "../dumbo" }
 logger = { path = "../logger" }
 vm-memory = { path = "../vm-memory" }
 utils = { path = "../utils" }
 net_gen = { path = "../net_gen" }
+polly = { path = "../polly" }
 rate_limiter = { path = "../rate_limiter" }
 virtio_gen = { path = "../virtio_gen" }
+
+[dev-dependencies]
+vmm-sys-util = ">=0.3.1"
```

Comment:
This looks like a leftover. We should use the local `utils` crate instead which is already declared as a dependency. Otherwise we might end up with multiple versions of vmm-sys-util.

The same comment applies to the vmm crate as well.

Reply:
fixed